src/validation_workflow/docker/validation_docker.py

```diff
@@ -56,15 +86,21 @@ def start_cluster(self) -> bool:
         return True
 
     def validation(self) -> bool:
-        # STEP 2 . inspect image digest betwwen opensearchproject(downloaed/local) and opensearchstaging(dockerHub)
-
-        self._OS_inspect_digest = InspectDockerImage(self.local_image_OS_id, self.args.OS_image, self.args.version)
-        self._OSD_inspect_digest = InspectDockerImage(self.local_image_OSD_id, self.args.OSD_image, self.args.version)
-
-        if self._OS_inspect_digest.inspect_digest() and self._OSD_inspect_digest.inspect_digest():
-            logging.info('Image digest is validated\n\n')
+        # STEP 2 . inspect image digest between opensearchproject(downloaed/local) and opensearchstaging(dockerHub)
```

Comment:
nit: typo in downloaed. Should be downloaded/local

Reply:
oops ./ thanks for the correction .. let me get that right in my next commit